### DISCUSSION TITLE
ESP32-c3 boards ???

### DISCUSSION THREAD

**Discussion by SimonVail** [joedevivo/vscode-circuitpython#85] [General]
Hi, 
I would like to know where are those boards ? 
I have a ESP32-C3-DevkitC-02 and I would like to know how to add them ?

**Comment by SimonVail** [comment]
@joedevivo ???

**Reply by joedevivo** [reply]
According to [this](https://docs.circuitpython.org/en/latest/ports/espressif/README.html), CircuitPython support for that board is in beta. The extension only currently supports boards listed [here](https://circuitpython.org/downloads), as it generates the board metadata and `board` module stubs from `adafruit/circuitpython`'s `ports` directory.

**Comment by u8915055** [comment]
Just wondering if there should be support of C3 boards?  Seems circuitpython is fully supported but i just cant seem to get this extension to recognize a C3 board.. or is there another board i should choose from the list that might be compatible?
